### PR TITLE
added a modal popup for mobile and desktop design

### PR DESCRIPTION
**Pull request by arabiu033**
- Add a modal popup for the design.
- Customized the modal with CSS
- generate HTML content using javascript function
- Fix linters

Files Modified
- stylr.css
- script.js

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -338,7 +338,213 @@ textarea {
...
+    display: initial;
+  }
+
+  .work-info {
+    display: grid;
+    height: 90%;
+    width: 30%;
+    grid-template-columns: 1fr 1fr 1.2fr 60px;
+    grid-template-rows: 30px 50% 2fr 20px 2fr;
+  }
```
Comment (KayLemba) [inline style.css:477]: In your about me section your content seem to be overlappin see image below to see what i mean.

This is your design: 

Screen 5

Stick to this design: 
Screen Shot 6
 
 You can fix ths issue by add the fooling styling your about-me-card class in your css like this
Example:

.about-me-card {
    grid-template: 1fr/1fr 1fr 1fr;
    gap: 3%;
    margin-top: 10vh;
    padding: 0 20%;
}

So in this case you need to adjust the margin-top to 10vh

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @arabiu033  👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉

- [x] No linter errors. 

- [x] GitHub flow is followed. 

- [x] Neat design UI

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

## Getting Started

[OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes git clone URL , cd project_name, etc 🙃 Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.
The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.
## Authors

[OPTIONAL] This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 👨🏻‍🎨
You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 1 above)
Comment (arabiu033) [reply style.css:477]: Thank you. adjusted

File: script.js [hunk 2]
```diff
@@ -35,4 +35,137 @@ exit.addEventListener('click', closeMenu);
...
+    title: 'Keeping track of hundreds of components',
+    technologies: {
+      t1: 'Ruby on rails',
+      t2: 'Css',
+      t3: 'JavScript',
+      t4: 'html',
+    },
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea`',
+    linklive: '#',
+    linkSrc: '#',
```
[the remark this answers is not in the document]
Comment (arabiu033) [reply script.js:55]: Done

File: index.html [hunk 3]
```diff
@@ -32,7 +32,7 @@
           <a href="#contact-link"><h2 class="flg">Contact</h2></a>
         </div>
       </div>
-      <!-- The main heading information on the page-->
+      <!-- The main heading information on the pages-->
       <section id="main">
```
[the remark this answers is not in the document]
Comment (arabiu033) [reply index.html:36]: Done

File: style.css [hunk 4]
```diff
@@ -338,7 +338,213 @@ textarea {
...
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
```
[the remark this answers is not in the document]
Comment (arabiu033) [reply style.css]: Restyle. thank you

File: style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (arabiu033) [reply style.css]: Restyle. thank you

File: style.css [hunk 5]
```diff
@@ -338,7 +340,232 @@ textarea {
...
+    display: initial;
+  }
+
+  .work-info {
+    display: grid;
+    height: 90%;
+    width: 70%;
+    grid-template-columns: 1fr 1fr 1fr 30px;
+    grid-template-rows: 30px 55% 10% 10% 25%;
+  }
```
Comment (CollinsTatang) [inline style.css:485]: - [ ] Please kindly check the `About me` section of your app, the content seem to be overlapping the `Get My Resume` button. Kindly fix it as requested by the previous reviewer.
### **Take A Look**
abouGet

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @arabiu033, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (arabiu033) [reply style.css:485]: Honestly, I don't know what's happening. It appears ok from my side. but I updated some changes. i hope its fixed
Screenshot_2022-09-07_21-29-40

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @arabiu033👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (CollinsTatang) [reply style.css:485]: It's good now 
### **Optional** 

- [ ] Please you can add functionality to make the button work

**Pull request merged**